backend/src/main/java/trn/logistics/knapsack/dto/SolutionRequest.java

```diff
@@ -0,0 +1,21 @@
+package trn.logistics.knapsack.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Collection;
+
+@Getter
+@Slf4j
+@AllArgsConstructor
+public class SolutionRequest {
+
+    private Long id;
```

Comment:
Shouldn't this be part of the return value and not of the request?
Otherwise the client handles the IDs and not the server/database